cmd/estuary-shuttle/rpc.go

```diff
@@ -319,61 +321,81 @@ func (s *Shuttle) handleRpcAggregateStagedContent(ctx context.Context, cmd *drpc
 	default:
 		return err
 	case nil:
-		// exists already
+		// if it was aggregated successfully but estuary was not notified,, resend message
+		if p.Active {
+			return s.resendPinComplete(ctx, p)
+		}
 		return nil
```

Comment:
nit: should we log a warning if we're hitting this `return nil` case? sounds like we'd only hit it if we're sending redundant aggregate cmds, but would be good to make sure we're not doing it

Reply:
yeah, I warning will be good here